content/en/monitors/monitor_types/anomaly.md

```diff
@@ -109,14 +109,18 @@ This example shows how each algorithm handles a new metric. `Robust` and `agile`
 
 {{< img src="monitors/monitor_types/anomaly/alg_comparison_new_metric.png" alt="algorithm comparison new metric" responsive="true" style="width:90%;">}}
 
-### API
+### Notifications
 
-Enterprise-level customers can create anomaly detection monitors using the [create-monitor API endpoint][8]. Add the `anomalies` function to the query:
+For detailed instructions on the **Say what's happening** and **Notify your team** sections, see the [Notifications][8] page.
+
+## API
+
+Enterprise-level customers can create anomaly detection monitors using the [create-monitor API endpoint][9]. Add the `anomalies` function to the query:
 ```text
 time_aggr(eval_window_length):anomalies(space_aggr:metric{tags}, 'basic/agile/robust', deviation_number, direction='both/above/below', alert_window='alert_window_length', interval=seconds, count_default_zero='true') >= threshold_value
```

Comment:
We should not be recommending that users build these monitor queries manually. We should guide them to use the "Export Monitor" button. There are values in the query string and the JSON options in the monitor definition that need to be set in tandem , and there are tricky interactions between some of the parameters. The UI will help users make the right choices. Manually building queries will lead to many poorly configured monitors and subsequent support tickets.